tools/js-tools/git-hooks/pre-push-hook.js

```diff
@@ -40,6 +40,54 @@ function checkChangelogFiles() {
 			)
 		);
 	} else {
+		try {
+			// Run the changelogger.
+			const autoChangelog = spawnSync( 'jetpack', [ 'changelog', 'add' ], {
```

Comment:
Doesn't work if you don't have `jetpack` in your path.
```suggestion
			const autoChangelog = spawnSync( 'pnpx', [ 'jetpack', 'changelog', 'add' ], {
```

Reply:
Really good catch here.